ios/FluentUI/Controls/IndeterminateProgressBarView.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Indeterminate progress bar. Since iOS's UIprogressView is always determinate, this is created for handling indeterminate scenarios.
+@objc(MSFIndeterminateProgressBarView)
+open class IndeterminateProgressBarView: UIView, ActivityViewAnimating {
+
+    /// The progress bar view should be hidden when animation stops if set to true. The default value is true.
+    @objc open var hidesWhenStopped: Bool = true
+
+    open override var intrinsicContentSize: CGSize {
+        return CGSize(width: UIView.noIntrinsicMetric, height: frame.height)
+    }
+
+    open override func sizeThatFits(_ size: CGSize) -> CGSize {
+        return CGSize(width: size.width, height: Constants.progressBarHeight)
+    }
+
+    @objc public override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.frame.size.height = Constants.progressBarHeight
+        autoresizingMask = .flexibleWidth
+        clipsToBounds = true
+    }
+
+    required public init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```

Comment:
Should use preconditionFailure instead of fatalError. fatalError (stupidly) encodes the absolute file path on the building machine in the binary output